plugins/plugin-languageserver/che-plugin-languageserver-ide/pom.xml

```diff
@@ -149,7 +149,7 @@
                 <version>${project.version}</version>
                 <executions>
                     <execution>
-                        <phase>process-sources</phase>
+                        <phase>generate-sources</phase>
```

Comment:
last time we are not able to release with ```<phase>generate-sources</phase>``` I think there is a compilation issue in this phase. Why do you want to change it? Can you compile this artifact in offline mode if you local is empty or doesn't contains ```che-plugin-languageserver-ide``` artifact